<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meli</class>
 <widget class="QWidget" name="meli">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="pb_meli">
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="l_meli">
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Mercado BÃºsqueda</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="le_meli"/>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="imagenes_productos_grid">
     <item row="0" column="0">
      <widget class="QLabel" name="label_imagen_producto_1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>i1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_imagen_producto_4">
       <property name="text">
        <string>i4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_imagen_producto_2">
       <property name="text">
        <string>i2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_imagen_producto_3">
       <property name="text">
        <string>i3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_imagen_producto_5">
       <property name="text">
        <string>i5</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QGridLayout" name="datos_productos_grid">
     <item row="0" column="2">
      <widget class="QLabel" name="label_producto_3">
       <property name="text">
        <string>p3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_producto_2">
       <property name="text">
        <string>p2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_producto_4">
       <property name="text">
        <string>p4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_producto_1">
       <property name="text">
        <string>p1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_producto_5">
       <property name="text">
        <string>p5</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
